src/index.ts

```diff
@@ -279,3 +283,15 @@ export default async function app(
 
   return app;
 }
+
+if (process.env.NODE_ENV === 'development') {
```

Comment:
That should be removed I guess?

Reply:
Yes, just for testing purposes. Let me add a comment.